function.js

```diff
@@ -0,0 +1,7 @@
+function buyPhone(phones, budget) {
+  console.log(`If you want buy ${phones.brand} ${phones.model} you should have:
+    not less than ${budget} $`);
+}
+
+export { buyPhone };
```

Comment:
please use named export here. You don't have to use curly braces around `buyPhone`